client/dashboard/me/notifications-sites/browser-notification-notice/index.tsx

```diff
@@ -0,0 +1,28 @@
+// eslint-disable-next-line no-restricted-imports
+import { localizeUrl } from '@automattic/i18n-utils';
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { Notice } from '../../../components/notice';
+
+export const BrowserNotificationNotice = () => {
+	return (
+		<Notice
+			variant="warning"
+			density="low"
+			actions={
+				<>
+					<Button
+						variant="link"
+						href={ localizeUrl( 'https://wordpress.com/support/troubleshoot-browser-block-chat/' ) }
```

Comment:
I don't think that link is related to this, it looks to be about different issues. 

Maybe it should be this instead if we want a simple URL?

```suggestion
						href={ localizeUrl( 'https://wordpress.com/support/notifications/browser-notifications/#troubleshooting' ) }
```

Maybe it would be best to keep the existing logic to display a different link based on your browser though:

https://github.com/Automattic/wp-calypso/blob/e43d88b62a09adbb9bb7f13cf45bb4e2e76eb5ed/client/me/notification-settings/push-notification-settings/index.jsx#L32

Reply:
I'm marking this one as unresolved since it is still an issue.

Reply:
Updated thanks [f581679](https://github.com/Automattic/wp-calypso/pull/105571/commits/f581679451cb85e074aa06572bb8801733808521)